jkube-kit/parent/pom.xml

```diff
@@ -736,6 +736,12 @@
         <version>${version.mockito}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-inline</artifactId>
```

Comment:
Very strange that I needed this in OpenshiftHelmPushMojoTest for the `org.mockito.Mockito.mockConstruction(HelmService.class);` thingy, but HelmPushMojoTest doesn't need it...

Reply:
I think you can avoid this by adding [`mockito-inline`](https://github.com/eclipse/jkube/tree/master/kubernetes-maven-plugin/plugin/src/test/resources/mockito-extensions) folder in `src/test/resources`

Reply:
Wauw, I never knew! Thanks for the tip! Implemented in e463ef777222592ff93e4520d555c38d92b24e25.